### PR TITLE
Feature/enhance alert

### PR DESCRIPTION
**Pull request by SylviaGonschior**

### CODE REVIEW

File: packages/components/src/components/alert/alert.css [hunk 1]
```diff
@@ -1,38 +1,85 @@
...
+  color: black;
+  margin: 3px;
+  left: 0.75rem;
+  z-index: 10;
+  padding: 1px;
+  border-radius: 3px;
+  text-align: center;
+}
+
+.close {
```
Comment (eeegor) [inline packages/components/src/components/alert/alert.css]: Please consider renaming `.close` to `.alert__close`

File: packages/components/src/components/alert/alert.tsx [hunk 2]
```diff
@@ -14,6 +15,13 @@ export class alert {
   @Prop() size?: string = '';
   @Prop() theme?: string = '';
   @Prop() variant?: string = '';
+  @Prop({reflectToAttr: true}) title: string;
+  @Prop({reflectToAttr: true}) opened: boolean;
+  @Prop() timeout?: number;
```
Comment (eeegor) [inline packages/components/src/components/alert/alert.tsx]: Please consider adding the defaultTimeout to the prop itself: `@Prop() timeout?: number = 500`;

File: packages/components/src/components/alert/alert.tsx [hunk 3]
```diff
@@ -25,11 +33,50 @@ export class alert {
...
+  @Method()
+  open() {
+    this.opened = true;
+  }
+
+  onCloseAlertWithTimeout = () => {
+    if (this.timeout !== undefined) {
+      setTimeout(this.onCloseAlert, this.timeout);
+    } else if (!this.withTimeout) {
+      let defaultTimeout = 500;
```
Comment (eeegor) [inline packages/components/src/components/alert/alert.tsx]: As the default timeout is set directly on the prop (prev comment), you can check if `withTimeout === true` and only then use the `timeout` prop

File: packages/components/src/components/alert/readme.md [hunk 4]
```diff
@@ -7,12 +7,27 @@
...
 
-| Property      | Attribute      | Description | Type     | Default |
-| ------------- | -------------- | ----------- | -------- | ------- |
-| `customClass` | `custom-class` |             | `string` | `''`    |
-| `size`        | `size`         |             | `string` | `''`    |
-| `theme`       | `theme`        |             | `string` | `''`    |
-| `variant`     | `variant`      |             | `string` | `''`    |
+| Property      | Attribute      | Description | Type      | Default     |
+| ------------- | -------------- | ----------- | --------- | ----------- |
+| `customClass` | `custom-class` |             | `string`  | `''`        |
```
Comment (eeegor) [inline packages/components/src/components/alert/readme.md]: Please consider adding `description` to the props, see `button` component for an example

File: packages/components/src/index.html [hunk 5]
```diff
@@ -25,12 +25,45 @@
...
-  <h3>Button</h3>
-  <t-button variant="primary">Hello</t-button>
-  <h3>Card</h3>
-  <t-card>A title</t-card>
+<h1>Components</h1>
+<h3>Alert</h3>
+<button
+  id="show-alert-btn">show alert
+</button>
+<t-alert
```
Comment (eeegor) [inline packages/components/src/index.html]: Please add examples for all major usecases the component can handle:
1. No timeout
2. Default timeout
3. Custom timeout

File: packages/components/src/index.html [hunk 6]
```diff
@@ -25,12 +25,45 @@
...
+
+
+<h3>Card</h3>
+<t-card>A title</t-card>
+
+
+<script>
+  const showAlert = document.querySelector('#show-alert-btn');
+  const alert = document.querySelector('t-alert');
+  showAlert.addEventListener('click', () => {
```
Comment (eeegor) [inline packages/components/src/index.html]: Can be simplified into a one-liner: ('click', () => !alert.opened && alert.open());

File: packages/components/src/components/alert/alert.css (hunk 1 above)
Comment (SylviaGonschior) [reply packages/components/src/components/alert/alert.css]: ok

**Pull request merged**